resource_estimator/src/system/optimization/tfactory_exhaustive.rs

```diff
@@ -360,6 +361,19 @@ impl TFactoryBuilder {
     }
 }
 
+impl Default for TFactoryBuilder {
```

Comment:
Should the Default be utilized somewhere in this PR? Or is a follow up PR expected?

Reply:
There is a follow up PR planned, in which this is used in a user example to construct a default T factory builder.